cumulusci/tasks/salesforce/tests/test_Deploy.py

```diff
@@ -43,10 +44,365 @@ def test_get_api__managed(self):
+
+            # add lwc component files not included in zip
+            for lwc_ignored_file in lwc_ignored_files:
+                with open(os.path.join(lwc_component_path, lwc_ignored_file), "w"):
+                    pass
+
+            # add lwc component sub-directory and files not included in zip
+            lwc_component_test_path = os.path.join(lwc_component_path, "__tests__")
+            os.mkdir(lwc_component_test_path)
+            with open(os.path.join(lwc_component_test_path, "test.js"), "w"):
+                pass
+
+            # add classes
+            classes_path = os.path.join(path, "classes")
+            os.mkdir(classes_path)
+            class_files = [
+                {
+                    "name": "MyClass.cls-meta.xml",
+                    "body": """<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>45.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
+""",
+                },
+                {"name": "MyClass.cls"},
+            ]
+            for class_file in class_files:
+                with open(os.path.join(classes_path, class_file.get("name")), "w") as f:
+                    if class_file.get("body") is not None:
+                        f.write(class_file.get("body"))
+
+            # add objects
+            objects_path = os.path.join(path, "objects")
+            os.mkdir(objects_path)
+            object_file_names = ["Account.object", "Contact.object", "CustomObject__c"]
+            object_file_names.sort()
+            for object_file_name in object_file_names:
+                with open(os.path.join(objects_path, object_file_name), "w"):
+                    pass
+
+            # add sub-directory of objects (that doesn't really exist)
+            objects_sub_path = os.path.join(objects_path, "does-not-exist-in-schema")
+            os.mkdir(objects_sub_path)
+            with open(os.path.join(objects_sub_path, "some.file"), "w"):
+                pass
+
+            # test
+            task = create_task(
+                Deploy,
+                {
+                    "path": path,
+                    "namespace_tokenize": "ns",
+                    "namespace_inject": "ns",
+                    "namespace_strip": "ns",
+                },
+            )
+
+            zip_bytes = io.BytesIO()
+            zipf = zipfile.ZipFile(zip_bytes, "w", zipfile.ZIP_DEFLATED)
+
+            with cd(path):
+                for file_to_pacakge in task._get_files_to_package(path):
+                    zipf.write(file_to_pacakge)
+                zipf.close()
+
+            zipf_processed = task._process_zip_file(zipfile.ZipFile(zip_bytes))
+            fp = zipf_processed.fp
+            zipf_processed.close()
+            expected = bytes_to_native_str(base64.b64encode(fp.getvalue()))
+
+            actual = task._get_package_zip(path)
+
+            self.assertEqual(expected, actual)
```

Comment:
@davisagli 
I'm getting CI test failures asserting the result of `_get_package_zip` equals the expected.  The tests pass locally.  It's hard to debug, and the test essentially creates test data then uses a copy of the production code.  Any suggestions?

Reply:
I'm trying to figure it out too. It's especially weird since it passed in the build for the 2nd-to-last commit on this branch, and the only thing that's changed since then is fixing spelling.

Reply:
I re-ran the CircleCI build and it passed this time. Which is mildly disturbing; I guess we'll see if the problem continues to show up.